pytorch_lightning/trainer/callback_config.py

```diff
@@ -53,10 +47,6 @@ def configure_checkpoint_callback(self, checkpoint_callback):
         if checkpoint_callback:
             checkpoint_callback.save_function = self.save_checkpoint
 
-        # if weights_save_path is still none here, set to current working dir
-        if self.weights_save_path is None:
```

Comment:
without this what will be the path? just none?

Reply:
no, it will still be default_root_dir, it's just getting set in the Trainer init now.